src/main/resources/application.properties

```diff
@@ -1,7 +1,7 @@
 # Oracle settings
-spring.datasource.url=${CHIPS_DATASOURCE_URL}
-spring.datasource.username=${CHIPS_DATASOURCE_USERNAME}
-spring.datasource.password=${CHIPS_DATASOURCE_PASSWORD}
+spring.datasource.url=jdbc:oracle:thin:@172.24.4.54:1521:CHIPSDEV
```

Comment:
Not sure this is right. I'd have thought these values should still get picked up from env config...